tests/org.jboss.tools.examples.ui.bot.test/pom.xml

```diff
@@ -16,16 +16,16 @@
 		<!-- <systemProperties>${integrationTestsSystemProperties} -->
 		<!-- -DimportTestDefinition=${importTestDefinition}</systemProperties> -->
 		<systemProperties>${integrationTestsSystemProperties} -DexamplesLocation=${project.build.directory}/${examplesFolder} -Dmaven.settings.path=${maven.settings.path} -Drd.config=${reddeer.config} -DspecificQuickstarts=${specificQuickstarts} -DdeployOnServer=${deployOnServer}</systemProperties>
-		<quickstartsURLWildFly>https://github.com/wildfly/quickstart/archive/14.0.1.Final.zip</quickstartsURLWildFly>
+		<quickstartsURLWildFly>https://github.com/wildfly/quickstart/archive/17.0.0.Final.zip</quickstartsURLWildFly>
 		<quickstartsURLEAP>http://download.eng.brq.redhat.com/released/JBoss-middleware/eap7/7.2.1/jboss-eap-7.2.1-quickstarts.zip</quickstartsURLEAP>
 		<quickstartsURLJavaEE>https://github.com/javaee-samples/javaee7-samples/archive/master.zip</quickstartsURLJavaEE>
 		<maven.settings.path>${project.build.directory}/classes/settings.xml</maven.settings.path>
-		<reddeer.config.wildfly14>${project.build.directory}/classes/servers/wildfly-14.json</reddeer.config.wildfly14>
+		<reddeer.config.wildfly17>${project.build.directory}/classes/servers/wildfly-17.json</reddeer.config.wildfly17>
```

Comment:
You can change name of variables in pom to just WildFly(without version number), so you don't need to update it on so many places when updating WildFly.

Reply:
Fixed.